modules/nf-core/glimpse2/phase/main.nf

```diff
@@ -0,0 +1,69 @@
+process GLIMPSE2_PHASE {
+    tag "$meta.id"
+    label 'process_medium'
+
+    beforeScript  """
+    if cat /proc/cpuinfo | grep avx2 -q
+    then
+        echo "Feature AVX2 present"
+    else
+        echo "Feature AVX2 not present on node"
+        exit 1
+    fi
+    """
+
+    conda "bioconda::glimpse-bio=2.0.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/glimpse-bio:2.0.0--hf340a29_0':
+        'quay.io/biocontainers/glimpse-bio:2.0.0--hf340a29_0' }"
+
+    input:
+        tuple val(meta), path(input), path(input_index), val(input_region), val(output_region), path(reference), path(reference_index), path(map), path(samples_file)
+        tuple val(meta2), path(fasta_reference)
```

Comment:
No fasta index needed here?

Reply:
I could add it true.
It's not necessary but it seems to be generated if not present.